Mu2eG4/geom/geom_2021_PhaseI_v02.txt

```diff
@@ -0,0 +1,121 @@
+// Updated to be used in the MDC2020 simulation campaign, based off Mu2eG4/geom/geom_2019_PhaseI_hayman_v2.txt
+// updates the OPA support structures to remove overlaps
+
```

Comment:
Why is OPA update a part of the PS endcap geometry PR?  Or is the comment here wrong?

Reply:
I've updated this comment to reflect this update, instead of the previous one.